### ISSUE TITLE
Add templates written and described by the community

### ISSUE THREAD

**Issue by chshersh** [haskell/actions#13]
Inspired by #7.

I would be happy to add the following templates written and described in detail by me and @vrom911:

* [Dead simple cross-platform GitHub Actions for Haskell](https://kodimensional.dev/github-actions) — simplest copy-pasteable config to build Haskell projects on Ubuntu, macOS and Windows using the latest 3 major GHC versions and both build tools Cabal and Stack
* [Haskell binaries release with GitHub Actions](https://vrom911.github.io/blog/github-actions-releases) — GitHub releases with optimized binaries for Haskell tools

I think a lot of people can benefit from more examples and described templates of using the Haskell setup action 🤗 Also, licensing rules for our templates are simpler (no need to mention anyone and worry about that).

**Comment by cartazio** [comment]
@chshersh thats a dick flex

**Comment by cartazio** [comment]
i actually checked with several intellectual property lawyers who do software license work and they thought it was very reasonable and not an onerous requirement.

**Comment by cartazio** [comment]
i totally agree more examples is better, but that last aside in your ticket description is just  not cool and needlessly detracts from the intent/goal! 

but yeah, more examples to share is great

**Comment by chshersh** [comment]
> in your ticket description is just not cool and needlessly detracts from the intent/goal!

@cartazio I didn't detract from the goal. My exact words:

> Also, licensing rules for our templates are simpler

Simpler licensing rules mean that maintainers can copy-paste the config directly to their README if they want to make it more accessible to people. At the same time, they can spend less time researching licensing rules and understanding the consequences of their actions.

Again, I'm not saying that this is better, I'm only presenting opportunities and options to consider. Your behaviour is inappropriate and extremely rude. I never attacked you or your work. I never said that your config is worse than mine, or that my config is better than yours, or that you have unreasonable licensing rules. But your reaction is very offending and off-putting for everyone.

**Comment by cartazio** [comment]
your remark was in reference to 
```
# NOTICE: permission is granted reuse/adapt this file if and only if this header section is preserved
# or replicated in a  DEDICATION file in the top level directory of a project repository OR some end user visible element along with
# other open source license notices
# this file was originally authored by carter schonwald,
# In Memoriam: Graham Gordon "Papa". 1925/11/17 - 2020/12/03. A warm loving grandparent, he is missed.
# permission is granted to reuse/adapt this in your own CI, as long as you include the
# original license  and copyright info and associated attributions and memorial
```
where the only difference from MIT/BSD licensing is the preservation of a memorial line for a recently deceased family member.

**Comment by chshersh** [comment]
@cartazio I'm sorry for your loss. And, as I mentioned before, I have no issues with your licensing rules, and I don't consider them unreasonable. Licensing rules for my template are different and don't require a header.

Also, my comment wasn't about this specific difference between your header and MIT/BSD license. I'm not using MIT or BSD licenses.

**Comment by cartazio** [comment]
I appreciate that very much, thank you

I added the header requirement to ensure the memoriam gesture is preserved properly

**Comment by chshersh** [comment]
@cartazio I saw your template earlier, before you've added memoriam, and I didn't notice the dedication last time I checked it. So, when I said:

> Also, licensing rules for our templates are simpler (no need to mention anyone and worry about that).

all I mean is that people using your template, need to include the original license and copyright info and associated attributions, mention author (because I was aware only about author). When in my case it's not required, that's why I called it simpler.

@jared-w I'm closing this issue, it's not necessary to add my templates and resources to the repository.
